lib/reporting/identity_verification_report.rb

```diff
@@ -107,6 +107,7 @@ def as_csv
       csv << ['Workflow completed - In-Person Pending', idv_final_resolution_in_person]
       csv << ['Workflow completed - Fraud Review Pending', idv_final_resolution_fraud_review]
       csv << []
+      csv << ['Users who did not pass fraud review', idv_fraud_rejected]
```

Comment:
`idv_fraud_rejected` is already present; it just wasn't included in the Daily IdV Report.

Reply:
"did not pass" sounds to me like it would include "fraud review pending" as well as "fraud review rejected", WDYT of making the title clearer here?
```suggestion
      csv << ['Rejected fraud review', idv_fraud_rejected]
```

Reply:
Clearer _and_ simpler. I love it.